src/board/audio/built-in-sounds.ts

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
This is a manual conversion from CODAL by me (not sure where this code ended up in MakeCode).